site/app/views/calendar/CalendarView.php

```diff
@@ -100,7 +100,9 @@ public function showCalendar(CalendarInfo $info, GlobalCalendarInfo $global_info
             "is_superuser" => $this->core->getUser()->getAccessLevel() === User::LEVEL_SUPERUSER,
             "colors" => $info->getColors(),
             "instructor_courses" => $this->core->getQueries()->getInstructorLevelUnarchivedCourses($this->core->getUser()->getId()),
-            "view_cookie" => isset($_COOKIE['view']) ? $_COOKIE['view'] : "month",
+            "view_cookie" => $_COOKIE['view'] ?? $_COOKIE['view'] ?? "month",
+            "month_cookie" => $_COOKIE['calendar_month'] ?? $_COOKIE['calendar_month'] ?? $month,
+            "year_cookie" => $_COOKIE['calendar_year'] ?? $_COOKIE['calendar_year'] ?? $year,
```

Comment:
The null coalescing operator returns the value on the left if it is not null, this can be simplified as follows:
```suggestion
            "view_cookie" => $_COOKIE['view'] ?? "month",
            "month_cookie" => $_COOKIE['calendar_month'] ?? $month,
            "year_cookie" => $_COOKIE['calendar_year'] ?? $year,
```